modules/single-node-asg/main.tf

```diff
@@ -21,12 +21,20 @@ locals {
   # To give us a short-hand refernce to the AZ
   az = data.aws_subnet.server-subnet.availability_zone
 
-  # The "name_prefix" we provide to the persistent-ebs module
-  data_volume_name_prefix = "${var.name_prefix}-${var.name_suffix}-data"
-
   # The `name_prefix` we provide to the `iam-instance-profile` module
   # The persistent-ebs module appends the AZ to the data node name, other modules don't do that
   name_prefix_with_az = "${var.name_prefix}-${var.name_suffix}-${local.az}"
+
+  data_volume_default = {
```

Comment:
why the `_default` in the name?

Reply:
It means the default value of the map. In variable, we cannot specify default values for list(map(any)). So this is the way to reach the same goal (along with the merge call below).